test/jest.setup-file.ts

```diff
@@ -56,7 +58,7 @@ async function mockImportKey(
     crypto.subtle.importKey = parentImportKey;
     try {
         if (format === 'jwk') {
-            return crypto.subtle.importKey(
+            return await crypto.subtle.importKey(
```

Comment:
what is the reason to await instead of returning a promise that would be awaited?

Reply:
From the linter [rule](https://typescript-eslint.io/rules/return-await/) docs: 
> When the return statement is in try...catch, awaiting the promise also allows the promise's rejection to be caught instead of leaving the error to the caller.